packages/gpl-core/src/constants.ts

```diff
@@ -24,7 +36,24 @@ export const GPLSESSION_PROGRAMS = {
   ),
 };
 
+export const GPL_NAMESERVICE_PROGRAMS = {
+  "mainnet-beta": new anchor.web3.PublicKey(
+    "5kWEYrdyryq3jGP5sUcKwTySzxr3dHzWFBVA3vkt6Nj5"
+  ),
+  devnet: new anchor.web3.PublicKey(
+    "5kWEYrdyryq3jGP5sUcKwTySzxr3dHzWFBVA3vkt6Nj5"
+  ),
+  localnet: new anchor.web3.PublicKey(
+    "5kWEYrdyryq3jGP5sUcKwTySzxr3dHzWFBVA3vkt6Nj5"
+  ),
+};
+
+export const GRAPHQL_ENDPOINTS_OLD = {
+  "devnet": "https://gum-indexer-devnet-lafkve5tyq-uc.a.run.app/v1/graphql",
+  "mainnet-beta": "https://gum-indexer-mainnet-lafkve5tyq-uc.a.run.app/v1/graphql",
+};
+
 export const GRAPHQL_ENDPOINTS = {
-  devnet: "https://gum-indexer-devnet-lafkve5tyq-uc.a.run.app/v1/graphql",
-  "mainnet-beta": "https://gum-indexer-mainnet-lafkve5tyq-uc.a.run.app/v1/graphql"
+  "devnet": "https://gum-indexer-smartprofile-devnet-lafkve5tyq-uc.a.run.app/v1/graphql",
```

Comment:
Not important right now, but I think we should give it a proper domain under gum